### PR TITLE
Issue2075 replace siunit and improve tuning pars

### LINKED ISSUE #2075: Replace all SIunits defined parameters with OBC compliant parameter definition & improve tuning parameterization documentation
- [x] Similarly to how the time parameters were refactored, refactor any parameters that were defined using `Modelica.SIunits`.
- [x] Add unit to tuning step parameter and make comment more clear
- [x] Propagate tuning parameter limits

### PR DESCRIPTION
**Pull request by milicag**
Closes #2075.

### CODE REVIEW

File: Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/MinimumFlowBypass/Subsequences/FlowSetpoint.mo [hunk 1]
```diff
@@ -0,0 +1,906 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.MinimumFlowBypass.Subsequences;
```
Comment (JayHuLBL) [inline Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/MinimumFlowBypass/Subsequences/FlowSetpoint.mo:1]: The package `OBC/ASHRAE/PrimarySystem/ChillerPlant/MinimumFlowBypass/Subsequences` should be deleted (manually may be).

File: Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/MinimumFlowBypass/package.order [hunk 2]
```diff
@@ -1,3 +1,4 @@
 Controller
 FlowSetpoint
+Subsequences
```
Comment (JayHuLBL) [inline Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/MinimumFlowBypass/package.order:3]: no `Subsequencs` package.

**Comment by JayHuLBL** [comment]
@milicag see my inline comments.

**Comment by milicag** [comment]
@JayHuLBL Thanks for your review, I'll now merge this in.

**Pull request merged**